views/partials/bodytemplate.pug

```diff
@@ -0,0 +1,81 @@
+body
+  h2 Welcome to the League of Legends API
+  div(class="mainBoi")
+    div(class="leftDocs")
+      div(class="leftTabs selected" id="getAll" onclick="selected('getAll')")
+        span GET
+        span /lol
+      div(class="leftTabs" id="getChampion" onclick="selected('getChampion')")
+        span GET
+        span /lol/champions
+      div(class="leftTabs" id="getRegion" onclick="selected('getRegion')")
+        span GET
+        span /lol/regions
+      div(class="leftTabs" id="getRole" onclick="selected('getRole')")
+        span GET
+        span /lol/roles
+      div(class="leftTabs" id="postChampion" onclick="selected('postChampion')")
+        span POST
+        span /lol/champions
```

Comment:
This route doesn't work as described

![image](https://user-images.githubusercontent.com/5775716/84073891-a288f400-a99f-11ea-98f1-1b47ad7e5647.png)